docs/about-mono/releases/5.2.0.md

```diff
@@ -27,6 +27,16 @@ To debug the assembly loading process, set the logging environment variables to
 
 The current default behavior is the same as in previous Mono versions. It is explicitly available with the `--assembly-loader=legacy` option.
 
+Mangler V1
+---------------------
+
+An initial implementation of the name mangler was created. It associates with each method a unique string that captures the unique attributes of the method. If two methods have the same mangled representation, they're the same method.
```

Comment:
I don't think a typical Mono user knows what the "mangler" is after reading this. Can you add some more details about why/where it is needed and when/how someone might want to use it?